lms/djangoapps/edraak_marketing_email/helpers.py

```diff
@@ -0,0 +1,17 @@
+from edraak_marketing_email.models import UnsubscribedUser
+
+
+def unsubscribe_from_marketing_emails(user):
+    is_unsubscribed = UnsubscribedUser.objects.filter(user=user).exists()
+    if is_unsubscribed:
+        return
+
+    UnsubscribedUser.objects.create(user=user)
```

Comment:
def unsubscribe_from_marketing_emails(user):
It's a little bit safer to use the following expression:

```suggestion
    UnsubscribedUser.objects.get_or_create(user=user)
```

It handles the `if` statement.